$NetBSD: patch-scripts_ZoneMinder_lib_ZoneMinder_ConfigData.pm.in,v 1.3 2026/07/30 12:36:00 gdt Exp $

Disable check-for-updates ("phone home") by default.  pkgsrc doctrine is
that phone-home behavior, unless opted into by the user, is a bug.

\todo Perhaps discuss with upstream, but this is a difference in
judgement rather than a clear bug.

--- scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in.orig	2018-12-08 14:22:36.000000000 +0000
+++ scripts/ZoneMinder/lib/ZoneMinder/ConfigData.pm.in
@@ -2693,7 +2693,7 @@ our @options = (
   },
   {
     name        => 'ZM_CHECK_FOR_UPDATES',
-    default     => 'yes',
+    default     => 'no',
     description => 'Check with zoneminder.com for updated versions',
     help        => q`
       From ZoneMinder version 1.17.0 onwards new versions are
